pkg/pubsub/pubsub.go

```diff
@@ -0,0 +1,82 @@
+/*
+ * MinIO Cloud Storage, (C) 2019 MinIO, Inc.
```

Comment:
Space after headling
```suggestion
 * MinIO Cloud Storage, (C) 2019 MinIO, Inc.
 
```